doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -95,35 +84,36 @@ nRF9160: Asset Tracker v2
 
 * Added:
 
-   * Support for :ref:`Bosch Software Environmental Cluster (BSEC) library <bosch_software_environmental_cluster_library>` driver .
-   * Support for indoor air quality (IAQ) readings retrieved from the BME680 sensor on Thingy:91. For more information, see the :ref:`asset_tracker_v2_sensor_module`.
-   * Support for QEMU x86 emulation.
+  * Support for :ref:`Bosch Software Environmental Cluster (BSEC) library <bosch_software_environmental_cluster_library>` driver.
+  * Support for Indoor Air Quality (IAQ) readings retrieved from the BME680 sensor on Thingy:91. For more information, see the :ref:`asset_tracker_v2_sensor_module`.
+  * Support for QEMU x86 emulation.
 
 * Updated:
 
   * For nRF Cloud builds, the configuration section in the shadow is now initialized during the cloud connection process.
   * Allow the :ref:`lib_nrf_cloud` library to handle modem FOTA updates if :kconfig:option:`CONFIG_NRF_CLOUD_FOTA` is enabled.
 
-nRF9160: Serial LTE Modem
+nRF9160: Serial LTE modem
 -------------------------
 
+* Added:
+
+  * #XDFUGET and #XDFURUN AT commands to support the cloud-to-nRF52 DFU service.
```

Comment:
I think the command names should go in code ticks, as on the application doc pages, no?

Reply:
Fixed